### ISSUE TITLE
More Documentation

### ISSUE THREAD

**Issue by changspencer** [lalonderodney/SegCaps#5]
Hello,

I'm trying to understand the network itself. Could we get some comments for the classes in capsule_layers.py?

Thanks!

**Comment by lalonderodney** [comment]
I agree there should be more documentation. Unfortunately I was very sick for nearly two weeks and was receiving many requests for the code so I pushed it out with less comments than I would have preferred. I will try to make a point to add more comments when I can find some free time. Busy life of a PhD student :)
